quizzes/pop-quiz-2.md

```diff
@@ -39,9 +52,13 @@
   ```
 
 8.  Define a attr_reader in code, and what does it do?
```

Comment:
0/2

Answer

```ruby
def attr_example
  @attr_example
end
```
It creates a method that returns the value of an instance variable.